src/app/components/GoogleSheetsInput.svelte

```diff
@@ -0,0 +1,179 @@
+<script>
+  import worker from "../spawn-worker";
+
+  import InputField from "./InputField.svelte";
+  import Button from "./Button.svelte";
+
+  var spreadsheetId = "";
+  var error = null;
+
+  // Client ID and API key from the Developer Console
+  var CLIENT_ID = process.env.G_CLIENT_ID;
```

Comment:
```suggestion
  const CLIENT_ID = process.env.G_CLIENT_ID;
```
`const` would be more appropriate here since we do not have this variable to be modified later. Same for the **API_KEY**, **DISCOVERY_DOCS** and **SCOPES**